vars/perfRegressionParallelPipeline.groovy

```diff
@@ -48,6 +48,10 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('email_recipients', 'qa@scylladb.com')}",
                    description: 'email recipients of email report',
                    name: 'email_recipients')
+
+            string(defaultValue: "false",
+                   description: 'Initialize cluster in parallel (false) or sequentially(true)',
+                   name: 'use_legacy_cluster_init')
```

Comment:
@roydahan Also i added new parameter to job, which allow to switch the old and new cluster initialization